### PR TITLE
Updating Scrutiny Rockon installation write-up

### LINKED ISSUE #383: Update Scrutiny Rock-On write-up
Following the recent move to Scrutiny's official docker image (https://github.com/rockstor/rockon-registry/pull/331), the write-up needs to be updated to reflect the new options required by this image.

### PR DESCRIPTION
**Pull request by Hooverdan96**
> To ease and accelerate the PR submission, please use the template below to provide all required information.
> Please see our "Contributing to Rockstor documentation" in our [Rockstor's documentation](http://rockstor.com/docs/contribute_documentation.html)

Fixes #383  

### Updating the scrutiny Rockon installation write-up since switching to official container.
text and screenshots updated.

### Checklist
- [X] With the proposed changes no Sphinx errors or warnings are generated.
- [X] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by Hooverdan96** [comment]
@phillxnet, @FroggyFlox not sure why the linkcheck failed yesterday. I manually tested this URL today and it came back with the web page in my browser: 

https://www.ibm.com/support/pages/node/807406 .

The error was supposedly located in this file:

https://raw.githubusercontent.com/rockstor/rockstor-doc/master/howtos/soho.rst:

https://github.com/rockstor/rockstor-doc/actions/runs/4813087032/jobs/8569166761?pr=428#step:5:407

File: interface/docker-based-rock-ons/scrutiny.rst [hunk 1]
```diff
@@ -195,14 +194,14 @@ installation.
...
-Step 5, installation in progress, you can close the wizzard.
+Step 5, with the installation in progress, close the wizard.
 
 .. image:: /images/interface/docker-based-rock-ons/scrutiny_install_step5.png
    :width: 100%
    :align: center
 
 
-Once the installation done, you can use the **Scrutiny UI** button to go to the
+Once the installation done, use the **Scrutiny UI** button to go to the
```
Comment (phillxnet) [inline interface/docker-based-rock-ons/scrutiny.rst:204]:
```suggestion
Once the installation finishes, use the **Scrutiny UI** button to go to the
```

Comment (phillxnet) [reply interface/docker-based-rock-ons/scrutiny.rst:204]: Minor grammatical suggestion.

**Review by phillxnet — APPROVED** [review]
@Hooverdan96 Thanks for tending to this write-up: much appreciated.

Plus it's nice to have in docs the simpler setup of our newer (upstream) docker image.

There's only one small suggestion I have, if you choose to accept it, but I think we can pop this in regardless. Let me know if you think it's worth popping in the suggestion so I can publish either way.

File: interface/docker-based-rock-ons/scrutiny.rst (hunk 1 above)
Comment (Hooverdan96) [reply interface/docker-based-rock-ons/scrutiny.rst]: Makes perfect sense. I think I did not touch that portion from the original commit, but, yes.

**Comment by Hooverdan96** [comment]
Linkcheck failed this time again on the ibm related link, as well as due to one of the the archived btrfs links that generates a 404. I assume, we will fix that based on the other issue described in #424

**Comment by phillxnet** [comment]
@Hooverdan96 thanks again for updating this write up. Much appreciated.

PR product PRODUCTION published.
https://rockstor.com/docs/interface/docker-based-rock-ons/scrutiny.html

File: interface/docker-based-rock-ons/scrutiny.rst (hunk 1 above)
Comment (phillxnet) [reply interface/docker-based-rock-ons/scrutiny.rst]: Yes, my apologies for scope creep on this one.

**Pull request merged**